### DISCUSSION TITLE
IDs mapping to other unique identifiers

### DISCUSSION THREAD

**Discussion by bshor** [openstates/issues#73] [Proposals & RFCs]
Hello,

Is there some canonical mapping out there between the OCD unique identifiers for individual legislators and those of other data sets? For example, followthemoney, Votesmart, Legiscan, Ballotpedia, and so forth?

**Comment by showerst** [comment]
The openstates-people schema allows for it with the other_identifiers key, but it would be a large amount of work to build and maintain due to all the hand-matching that would be required to get 100% coverage.

If you're interested if trying to do some, we'd take the pull request =). 

Votesmart has https://github.com/votesmart/political-id-match which maps a couple of other third-party systems, so if you could match to one of those ids you'd get the others for free.

**Comment by bshor** [comment]
Got it. I was wondering if in some distant corner the work was already done by OpenStates.

I know about political id match but there aren't any OpenStates legacy or new identifiers :(

Yes, I hope to at some point to upload what I've found!

**Reply by brierjon** [reply]
Have you found anything else? Might take a look at Wikidata related proposal #645.

**Comment by bshor** [comment]
One more questions on the new OS OCD id's. They aren't displayed anywhere on the OpenStates site itself, are they?

For example, say Akasha Lawrence Spence (OR). Her OS id is

ocd-person/226307b4-2a1c-426d-acc5-6a74225521c9

and her OS web page is:

https://openstates.org/person/akasha-lawrence-spence-12szkbqQxxldXgCglPQk6z

But the OCD id isn't mentioned in the URL or the page itself. Is this by design?

**Comment by showerst** [comment]
Those IDs are primarily meant to be used by programmers, so there's really no reason to display them to end users. Is there a use case you had in mind where viewing them is helpful?

**Comment by jamesturk** [comment]
I'll add that the API contains the canonical URL for the cases where people
want to go the other way.
